contrib/drivers/lvm/lvmmetrics.go

```diff
@@ -0,0 +1,56 @@
+package lvm
+
+import (
+	"fmt"
+	"github.com/opensds/opensds/pkg/model"
+)
```

Comment:
This should be below the copyright header.

Reply:
fixed